extensions/connect-to-vpn/package.json

```diff
@@ -7,16 +7,24 @@
   "author": "rasmusbe",
   "contributors": [
     "sato11",
-    "stonerl"
+    "stonerl",
+    "vyacheslav"
   ],
   "license": "MIT",
   "commands": [
     {
-      "name": "index",
+      "name": "view",
```

Comment:
Can we keep the old name here? It's the unique ID of the command where ranking, aliases and hotkeys are saved

Reply:
Definitely! Wasn't aware it works like that. I'll make the change this evening 👍🏻

Reply:
Done ✅